src/PotentialCalculation/SuccessiveOverRelaxation/CPU_update.jl

```diff
@@ -0,0 +1,33 @@
+"""
+    function update!(   
+        pssrb::PotentialCalculationSetup{T}; 
+        ::Nothing, # these to unused arguments are used such that the method
+        ::Any;     # is similar to the GPU method for it.
+        use_nthreads::Int = Base.Threads.nthreads(), 
+        depletion_handling::Val{depletion_handling_enabled} = Val{false}(), 
+        is_weighting_potential::Val{_is_weighting_potential} = Val{false}(),
+        only2d::Val{only_2d} = Val{false}()
+)::Nothing where {T, depletion_handling_enabled, only_2d, _is_weighting_potential}
+
+This function performs one iteration of the SOR. One iteration consists out of 4 steps:
+
+    1) Iterate in parallel over all even points and update their potential. 
+    2) Apply the boundary conditions at the ends of the grid for all even points. 
+    3) Iterate in parallel over all odd points and update their potential. 
+    2) Apply the boundary conditions at the ends of the grid for all odd points. 
+"""
+@inline function update!(   
+    pssrb::PotentialCalculationSetup{T},
+    ::Nothing, # these to unused arguments are used such that the method
```

Comment:
Typo: two

Reply:
This typo is still in there.

Reply:
Ah, ups. Just corrected the one in the docstring.